simpleSequential/makeCoordinateTuple.py

```diff
@@ -14,3 +14,5 @@
 # Ex:
 #   makeCoordinateTuple(10, 1) # return (10, 1)
 #   makeCoordinateTuple(-5, 3) # return (-5, 3)
+def makeCoordinateTuple(x: int, y: int) -> tuple:
```

Comment:
Wrong output type. Please see the images that I sent you a few day earlier to know how to declare the output type correctly.

Reply:
Did you check this one? The data type is wrong.

Reply:
Acknowledge.